### PR TITLE
constructor pattern examples

### PR DESCRIPTION
**Pull request by Motopshyco**
This PR is for the examples of the constructor pattern

### CODE REVIEW

File: katas/design-patterns/constructor-pattern/carexample.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function Car(model, year, miles) {
```
Comment (daniel33gomez) [inline katas/design-patterns/constructor-pattern/carexample.js:1]: please add a space before the Car to maintain the same consistency as eslint

File: katas/design-patterns/constructor-pattern/fruitexample.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Fruit {
+    constructor(name, taste) {
```
Comment (daniel33gomez) [inline katas/design-patterns/constructor-pattern/fruitexample.js:2]: space after the constructor word

File: katas/design-patterns/constructor-pattern/fruitexample.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        this.name = name;
+        this.taste = taste;
+    }
+
+    getTaste() {
```
Comment (daniel33gomez) [inline katas/design-patterns/constructor-pattern/fruitexample.js:7]: space after getTaste

File: katas/design-patterns/constructor-pattern/informationexample.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+function information(name, age, gender, career) {
```
Comment (daniel33gomez) [inline katas/design-patterns/constructor-pattern/informationexample.js:1]: space before ()

**Review by daniel33gomez — CHANGES_REQUESTED** [review]
please add a space after you declare a function

File: katas/design-patterns/constructor-pattern/carexample.js (hunk 1 above)
Comment (Sam-sta) [reply katas/design-patterns/constructor-pattern/carexample.js:1]: Daniel, eslint ask for space before the braces, not in that parentesis

File: katas/design-patterns/constructor-pattern/fruitexample.js (hunk 2 above)
Comment (Sam-sta) [reply katas/design-patterns/constructor-pattern/fruitexample.js:2]: Same here as i said before, eslint doesn't require you to use space between the function declaration and the parenthesis

**Review by daniel33gomez — APPROVED** [review]

**Review by zkpain — APPROVED** [review]

**Review by Sam-sta — APPROVED** [review]

**Pull request merged**